flowbased-computation/flowbased-computation-impl/src/main/java/com/farao_community/farao/flowbased_computation/impl/LoopFlowExtensionInCrac.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.flowbased_computation.impl;
```

Comment:
Not sure that the location of this file is appropriate

Reply:
It depends on GlskProvider which is in flowbased_computation. 
Any suggestions on where we should put this Extension without making major refactoring?
Otherwise we can do a refactoring on flowbased and data etc..

Reply:
yes but GlskProvider is in flowbased_computation_api so you can use it from another module. As we merged in the master you can create a new module in /data/crac/crac-loopflow-extensions and put your classes there I think.

Reply:
Thanks!
Moved to new module.